From 92cbd983f22df2fd3bdc94082cbd0f3ea7c144bf Mon Sep 17 00:00:00 2001
From: Ding Yunliang <yunliang.ding@intel.com>
Date: Tue, 7 Jul 2015 15:38:59 +0800
Subject: [PATCH 443/462] atomisp: WA force DFS freq to 320MHz for 720P

WA on CHT, should have impact on both rear and front
sensor who have SDV enabled.

Change-Id: I4e4d5d1ff524441161594122950f817f7be6ff8f
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-40960
Signed-off-by: Ding Yunliang <yunliang.ding@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/63743
Approver: Bin B Han <bin.b.han@intel.com>
Reviewed-by: Bin B Han <bin.b.han@intel.com>
Build: IMIN (SC)
Validation-Android: Wei Tao <wei.tao@intel.com>
Integrator: WeiX Meng <weix.meng@intel.com>
Build: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Maintainer: WeiX Meng <weix.meng@intel.com>
---
 .../drivers/media/pci/atomisp2/atomisp_driver/atomisp_dfs_tables.h | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_dfs_tables.h b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_dfs_tables.h
index 34ce2a5..f7cedcd 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_dfs_tables.h
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_dfs_tables.h
@@ -336,6 +336,13 @@ static const struct atomisp_freq_scaling_rule dfs_rules_cht[] = {
 		.run_mode = ATOMISP_RUN_MODE_PREVIEW,
 	},
 	{
+		.width = 1280,
+		.height = 720,
+		.fps = ISP_FREQ_RULE_ANY,
+		.isp_freq = ISP_FREQ_320MHZ,
+		.run_mode = ATOMISP_RUN_MODE_SDV,
+	},
+	{
 		.width = ISP_FREQ_RULE_ANY,
 		.height = ISP_FREQ_RULE_ANY,
 		.fps = ISP_FREQ_RULE_ANY,
-- 
1.9.1

